core/src/main/java/org/mskcc/cbio/portal/web_api/GetClinicalData.java

```diff
@@ -149,9 +151,11 @@ public static JSONObject generateJson(List<ClinicalData> clinicals) throws DaoEx
             attrIds.add(c.getAttrId());
 //            }
         }
-        
+
         for (ClinicalAttribute attr : DaoClinicalAttributeMeta.getDatum(attrIds)) {
```

Comment:
Maybe we should change DaoClinicalAttributeMeta.getDatum to accept study id?

Reply:
Done.